Source/Controller/LaunchViewController.swift

```diff
@@ -108,7 +108,8 @@ class LaunchViewController: UIViewController {
 
         // transition to main app UI
         // note that delay which is to help the loginAndSync() call get content
-        DispatchQueue.main.asyncAfter(deadline: .now() + 3) {
+        // removing delay... let syncing happen in the background
+        DispatchQueue.main.asyncAfter(deadline: .now()) {
```

Comment:
Just use DispatchQueue.main.async {

}
it's more efficient than asyncAfter(.now()) per the docs

Reply:
Sounds good, my lack of knowledge of swift showing through.